app/docs/0.10.x/proxy.md

```diff
@@ -4,29 +4,26 @@ title: Proxy Reference
 
 # Proxy Reference
 
-As you may already know, Kong listens for traffic on three ports, which by
+Kong listens for traffic on three ports, which by
 default are:
 
 - `:8001` on which the [Admin API][API] used to configure Kong listens.
 - `:8000` on which Kong listens for incoming HTTP traffic from your clients, and 
-forwards it to your upstream services. This is the port that interests us in 
-this guide.
+forwards it to your upstream services. **This is the port that interests us in 
+this guide.**
 - `:8443` on which Kong listens for incoming HTTPs traffic. This port has a
 similar behavior as the `:8000` port, except that it expects HTTPs traffic. This
 port can be disabled via the configuration file.
 
-We will cover routing capabilities of Kong by explaining in details how incoming 
+In this document we cover routing capabilities of Kong by explaining in detail how incoming 
 requests on port `:8000` are proxied to a configured upstream service depending
 on their headers, URI, and HTTP method.
 
-It is worth pointing out that both of these ports can be configured. See the
-[Configuration Reference] for more details.
```

Comment:
I guessed that most readers would assume ports are configurable, and that most readers wouldn't want to configure ports at this point of reading the docs, and if they wanted to find out how to configure ports, they could easily do so by searching the docs. I don't have a strong preference here.